index.js

```diff
@@ -590,12 +618,7 @@ AFRAME.registerComponent('environment', {
         emissiveMap: this.gridTexture
       };
 
-      // use .shading for A-Frame < 0.7.0 and .flatShading for A-Frame >= 0.7.0
-      if (new THREE.Material().hasOwnProperty('shading')) {
-        this.groundMaterialProps.shading = this.environmentData.flatShading ? THREE.FlatShading : THREE.SmoothShading;
-      } else {
-        this.groundMaterialProps.flatShading = this.environmentData.flatShading;
-      }
```

Comment:
We only support aframe 1.3.0 and above with the current code related to geometries, so we can remove this material check.